code/web/src/modules/StyleSurvey/StyleSurvey.js

```diff
@@ -139,28 +219,36 @@ class StyleSurvey extends Component {
 
               <Grid justifyCenter={true} >
                 {/* if the answers.length === 0 */}
-              <Button style={{ margin: '1.5em 1.5em 1.5em 2.5em', height: '37px' }} theme="secondary">
+              {this.state.currentIndex !== 0 && <Button style={{ margin: '1.5em 1.5em 1.5em 2.5em', height: '37px' }} theme="secondary" onClick={() => this.navigateSurvey('back')}>
                 Back
-              </Button>
+              </Button>}
```

Comment:
I think we can wrap the buttons in a div container and give it a width property so the buttons don't move after being removed??

just an idea.